test/jdk/javax/swing/JTree/6263446/bug6263446.java

```diff
@@ -128,7 +144,6 @@ private static void click(int times) {
         for (int i = 0; i < times; i++) {
             robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
             robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
-            robot.waitForIdle();
```

Comment:
Why do we remove it here and then add it after each call of this "click" method? why not move to the end of the method?

Reply:
I wanted to keep it same as JTable/626344 one and it worked in the OCI systems...I do not want to change it/optimise it with the access to test in these OCI systems being limited...